owtf/db/url_manager.py

```diff
@@ -105,6 +105,19 @@ def add_to_db(self, url, visited, found=None, target_id=None):
             self.db.session.merge(models.Url(target_id=target_id, url=url, visited=visited, scope=scope))
             self.db.session.commit()
 
+    def get_num_urls(self, target_id=None):
+        count = self.db.session.query(models.Url(target_id=target_id)).count()
+        return count
+
+    def add_urls_start(self):
+        self.num_urls_before = self.get_num_urls()
+
+    def add_urls_end(self):
+        num_urls_after = self.get_num_urls()
+        msg = str(num_urls_after - self.num_urls_before)+" URLs have been added and classified"
+        logging.info(msg)
```

Comment:
Is this debug output?

Reply:
This is not, but the function is not used anyway. I am removing it.